app/policies/api/v1/plans_policy.rb

```diff
@@ -16,9 +16,8 @@ class Scope
         # User (non-admin) can view: any personal or organisationally_visible
         # User (admin) can view: all from users of their organisation
         def resolve
-          ids = Plan.publicly_visible.pluck(:id)
-          ids += plans_for_client if @user.is_a?(ApiClient)
-          ids += plans_for_user if @user.is_a?(User)
+          ids = plans_for_client if @user.is_a?(ApiClient)
```

Comment:
lets consolidate these to: `ids = @user.is_a?(ApiClient) ? plans_for_client : plans_for_user`
and then a simple `return [] unless @user.present?` in those two 'plans_for_x' methods to be safe

Reply:
Will make change soon.

Reply:
@briri updated with change requested.